web/src/components/Page.tsx

```diff
@@ -36,7 +36,7 @@ const Page = ({
   thumbnailSrcSet,
   content,
   lastUpdate,
-  showLastUpdateText = true,
+  showLastUpdateText = false,
```

Comment:
I think you shouldn't just change this. This changes behavior in a lot of other use cases. Please revert this and instead pass `false` as a prop in the specific case you don't want to show it.